src/components/button/button.js

```diff
@@ -179,6 +179,7 @@ function MdButtonDirective($mdButtonInkRipple, $mdTheming, $mdAria, $timeout) {
       });
     }
 
+    element.append('<div class="md-touch-target"></div>');
```

Comment:
I'm having a bit of difficulty doing this because of the `transclude` option. Anything I append there gets removed. I've tried this, with no success:

```
compile: function (element, attr) {
      element.append('<div class="md-touch-target"></div>');
      return postLink;
    }
```

It seems like in `$compile` `$transclude` has been deprecated as well. It's advised to do interactions it in postLink instead.

>Note: The transclude function that is passed to the compile function is deprecated, as it e.g. does not know about the right outer scope. Please use the transclude function that is passed to the link function instead.

https://docs.angularjs.org/api/ng/service/$compile